internal/tracinginterceptor/interceptor.go

```
@@ -186,17 +186,22 @@ func (i *Interceptor) HandleStream(s *transport.ServerStream, h transport.Stream
 		Headers:   req.Meta.Headers,
 		Transport: req.Meta.Transport,
 	}
-
 	extractOpenTracingSpan := &transport.ExtractOpenTracingSpan{
 		ParentSpanContext: parentSpanCtx,
 		Tracer:            i.tracer,
-		TransportName:     s.Request().Meta.Transport,
+		TransportName:     req.Meta.Transport,
 		StartTime:         time.Now(),
 		ExtraTags:         commonTracingTags,
 	}
 	_, span := extractOpenTracingSpan.Do(s.Context(), transportRequest)
 	defer span.Finish()
-	err := h.HandleStream(s)
+
+	tracedRaw := &tracedServerStream{
+		serverStream: s,
+		span:         span,
+	}
+	wrapped, _ := transport.NewServerStream(tracedRaw)
```

we should not ignore error here. Can you please handle it properly. Also add test case for it.

---

addressed, thanks!